app/jobs/sms_otp_sender_job.rb

```diff
@@ -0,0 +1,20 @@
+class SmsOtpSenderJob < ActiveJob::Base
+  queue_as :sms
+
+  def perform(code:, phone:, otp_created_at:)
+    otp_invalid?(otp_created_at) || send_otp(TwilioService.new, code, phone)
```

Comment:
I find this easier to read than 

```
return if otp_invalid?(otp_created_at)
send_otp(TwilioService.new, code, phone)
```

but I still prefer this. for clarity: (which our code climate config frowns upon)

```
if otp_invalid?(otp_created_at)
  send_otp(TwilioService.new, code, phone)
end
```